OFEP-metric-hooks.md

```diff
@@ -0,0 +1,67 @@
+## OFEP-hook-for-metrics
+
+## State: PENDING REVIEW
+
+This OFEP propose to introduce OpenFeature hook for OpenTelemetry metrics.
+
+## Background
+
+We already have OpenTelemetry Span support through hooks. Similarly, we can provide a dedicated hook for 
+OpenTelemetry 
+metrics. Providing telemetry data out of the box will make OpenFeature attractive for both users and vendors. 
+
+## Proposal
+
+The proposal here is to define a set of metrics that can be used with different hook stages. For example, error metrics 
+can be collected at the hook's `error` stage. Going with this background, I propose the following metrics.
+
+### evaluationRequests
+
+A counter[1] based metric to calculate the number of flag evaluation requests. This is recorded at `before` stage of 
+the hook.
+
+### evaluationSuccess
```

Comment:
I think it would help to mention some practical examples for some of these metrics. I think for `evaluationSuccess` in particular, they are basically what vendors refer to as "impressions". You might want to mention some such use-cases.

Not required for my approval though.

Reply:
Added a description to highlight usage of the metrics